.azure-pipelines.yml

```diff
@@ -162,3 +162,27 @@ jobs:
         condition: "and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/tags/'))"
         env:
           NPM_TOKEN: $(NPM_TOKEN)
+---
+variables:
```

Comment:
I think it would make sense to have this test on GH actions side where documentation is generated. See https://github.com/lensapp/lens/blob/master/.github/workflows/main.yml . Or if kept here then run only if docs are changed.

Reply:
Makes sense, will try using GH actions.